crates/egui-winit/Cargo.toml

```diff
@@ -61,6 +61,7 @@ puffin = { version = "0.14", optional = true }
 serde = { version = "1.0", optional = true, features = ["derive"] }
 
 webbrowser = { version = "0.8.3", optional = true }
+raw-window-handle = "0.5.0"
```

Comment:
Move under the native dependency list below (`[target.'cfg(not(target_arch="wasm32"))'.dependencies]`)